### PR TITLE
add Dockerfile 

### PR DESCRIPTION
**Pull request by riderx**
This add allow developper to build docker machine with lemverse in it .

I choose docker base image from https://github.com/disney/meteor-base 
it allow us to build into the docker. and produce output image without build tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,6 +390,15 @@ server {
...
 
+## Build docker machine
+
+### production
+
+docker buid . -t l3mpire/lemverse:latest
+
+### dev
+
+docker buid -f Dockerfile.dev . -t l3mpire/lemverse:latest
```
Comment (DblK) [inline README.md:401]: I tend to use different tag from source to not overwrite official images.
```suggestion
To build from you source without having to install anything, you can run the following command:  
`docker buid -f Dockerfile.dev . -t l3mpire/lemverse:dev`
```

**Review by DblK — CHANGES_REQUESTED** [review]
First, thank you so much for your PR! I was waiting for a docker version of lemverse myself!

For the content, I made some few suggestions (so easy for you to add them), feel free to answer if you have additional questions.

PS: We will squash the commits so do not make any rebase or something else ;)

File: README.md [hunk 2]
```diff
@@ -390,6 +390,15 @@ server {
 
 Once you have done it, the subsequent deployments will be done using the command `./deploy`.
 
+## Build docker machine
+
+### production
+
+docker buid . -t l3mpire/lemverse:latest
```
[the remark this answers is not in the document]
Comment (goyan) [reply README.md:397]: typo: 'buid'

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
+FROM geoffreybooth/meteor-base:2.3.6
```
[the remark this answers is not in the document]
Comment (riderx) [reply Dockerfile]: Agree, and fixed

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# Runtime dependencies; if your dependencies need compilation (native modules such as bcrypt) or you are using Meteor <1.8.1, use app-with-native-dependencies.dockerfile instead
+RUN apk --no-cache add \
+		bash \
+		ca-certificates \
+        imagemagick \
+        graphicsmagick
+
+# Copy in entrypoint
+COPY --from=0 $SCRIPTS_FOLDER $SCRIPTS_FOLDER/
```
[the remark this answers is not in the document]
Comment (riderx) [reply Dockerfile]: Done !

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+# Copy in app bundle
+COPY --from=0 $APP_BUNDLE_FOLDER/bundle $APP_BUNDLE_FOLDER/bundle/
```
[the remark this answers is not in the document]
Comment (riderx) [reply Dockerfile]: Done !

File: Dockerfile.dev [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+		ca-certificates \
+        imagemagick \
+        graphicsmagick
+
+
+WORKDIR /opt/src
+
+RUN chown -Rh root .meteor/local
+
+CMD ["meteor", "--settings", "settings-dev.json", "--port", "3000"]
```
[the remark this answers is not in the document]
Comment (riderx) [reply Dockerfile.dev]: Done !

**Comment by riderx** [comment]
@DblK thanks for the feedback i agree 100%, I updated all

**Comment by riderx** [comment]
You can find image i have create here : https://hub.docker.com/repository/docker/riderx/lemverse

**Review by DblK — APPROVED** [review]

**Pull request merged**